Source/AudioPlayer.swift

```diff
@@ -12,20 +12,57 @@ public protocol AudioPlayerDelegate {
     func playerDidUpdatePosition(seconds: Float)
 }
 
+public struct AudioFileItem {
```

Comment:
All of the computed properties here are calculated every time you try to access them. That could cause unnecessary computing, since all of the other properties are let (so they are never changed). A better approach would be to set them in the init method, or define them as lazy vars.